client/v2/go.mod

```diff
@@ -160,10 +161,12 @@ require (
 replace github.com/cosmos/cosmos-sdk => ./../../
 
 replace (
+	cosmossdk.io/x/auth => ./../../x/auth
 	cosmossdk.io/x/bank => ./../../x/bank
 	cosmossdk.io/x/distribution => ./../../x/distribution
 	cosmossdk.io/x/gov => ./../../x/gov
 	cosmossdk.io/x/mint => ./../../x/mint
+	cosmossdk.io/x/protocolpool => ./../../x/protocolpool
```

Comment:
Is this a necessary change ?

Reply:
Yeah, `go mod tidy` was failing without it.